rogue-thi-app/data/calendar.json

```diff
@@ -9,220 +9,220 @@
-    "begin": "2022-07-02",
-    "end": "2022-08-07"
-  },
-  {
-    "name": "Erweiterter Prüfungszeitraum",
-    "begin": "2022-07-02",
-    "end": "2022-07-08"
-  },
-  {
-    "name": "Prüfungszeitraum",
-    "begin": "2022-07-09",
-    "end": "2022-07-19"
-  },
-  {
-    "name": "Notenbekanntgabe",
-    "begin": "2022-07-29T13:00",
-    "hasHours": true
+    "name": {
+      "de": "Semesterferien",
+      "en": "Semester break"
+    },
+    "begin": "2023-08-01",
+    "end": "2023-09-30"
   },
   {
-    "name": "Semesterferien",
-    "begin": "2022-07-30",
-    "end": "2022-09-30"
+    "name": {
+      "de": "Vorlesungsbeginn und Semesterstart",
+      "en": "Start of lectures and semester"
+    },
+    "begin": "2023-10-02"
   },
   {
-    "name": "Vorlesungsbeginn und Semesterstart",
-    "begin": "2022-10-04"
+    "name": {
+      "de": "Vorlesungsfrei",
+      "en": "Lecture-free"
+    },
+    "begin": "2023-10-03"
   },
   {
-    "name": "Vorlesungsfrei",
-    "begin": "2022-10-31"
+    "name": {
+      "de": "Vorlesungsfrei",
+      "en": "Lecture-free"
+    },
+    "begin": "2023-10-31"
   },
   {
-    "name": "Prüfungsanmeldung",
-    "begin": "2022-11-04",
-    "end": "2022-11-14"
+    "name": {
+      "de": "Prüfungsanmeldung",
+      "en": "Exam registration"
+    },
+    "begin": "2023-11-06",
+    "end": "2023-11-16"
   },
   {
-    "name": "Prüfungsabmeldung",
-    "begin": "2022-11-04",
-    "end": "2023-01-11"
+    "name": {
+      "de": "Prüfungsabmeldung",
+      "en": "Exam withdrawal"
+    },
+    "begin": "2023-11-06",
+    "end": "2024-01-11"
   },
   {
-    "name": "Vorlesungsfrei (Weihnachten)",
-    "begin": "2022-12-24",
-    "end": "2023-01-08"
+    "name": {
+      "de": "Vorlesungsfrei (Weihnachten)",
+      "en": "Lecture-free (Christmas)"
+    },
+    "begin": "2023-12-24",
+    "end": "2024-01-07"
   },
   {
-    "name": "Notenmeldung Prädikate (ZV)",
-    "begin": "2023-01-09T09:00",
+    "name": {
+      "de": "Notenmeldung Prädikate (ZV)",
+      "en": "Grade submission predicates (ZV)"
+    },
+    "begin": "2024-01-09T9:00",
     "hasHours": true
   },
   {
-    "name": "Erweiterter Prüfungszeitraum",
-    "begin": "2023-01-19",
-    "end": "2023-01-25"
+    "name": {
+      "de": "Erweiterter Prüfungszeitraum",
+      "en": "Additional exam period"
+    },
+    "begin": "2024-01-19",
+    "end": "2024-01-25"
   },
   {
-    "name": "Prüfungszeitraum",
-    "begin": "2023-01-26",
-    "end": "2023-02-04"
+    "name": {
+      "de": "Ende der Vorlesungszeit",
+      "en": "End of lecture period"
+    },
+    "begin": "2024-01-25"
   },
   {
-    "name": "Ende der Vorlesungszeit",
-    "begin": "2023-01-25"
+    "name": {
+      "de": "Prüfungszeitraum",
+      "en": "Exam period"
+    },
+    "begin": "2024-01-26",
+    "end": "2024-02-05"
   },
   {
-    "name": "Notenmeldung",
-    "begin": "2023-02-09T09:00",
+    "name": {
+      "de": "Notenmeldung",
+      "en": "Grade submission"
```

Comment:
"Official translation" would be "Recording grades" but I don't know about this one.